python/lsst/ctrl/stats/records/record.py

```diff
@@ -60,6 +64,24 @@ def __init__(self, year, lines):
             print(lines[0])
             sys.exit(10)
 
+    def getTimestamp(self):
+        return self.timestamp
+
+    def setTimestamp(self, timestamp):
+        self.timestamp = timestamp
+
```

Comment:
Turns out these were left over from a previous change in this ticket, so I'm deleting them.  They've never made it to the main branch so they're safe to get rid of.